File: ex2.2.py
def le():
    n1=int(raw_input('Digite o primeiro valor: '))
    n2=int(raw_input('Digite o segundo valor: '))
    return n1,n2
def imp(n1,n2):
    for i in range(n1,n2+1):
        print "{:1d}\n".format(i)

n1,n2=le()
imp(n1,n2)

File: ex2.2.c
#include <stdio.h>

int main(void){
	int n1, n2;
	printf ("Digite o primeiro numero: ");
	scanf ("%d", &n1);
	printf ("Digite o segundo numero: ");
	scanf ("%d", &n2);
	int i=n1;
	while (i<=n2){
		printf ("%d\n", i);
		i+=1;
	}
	return 0;
}
